app/controllers/webauthn_verifications_controller.rb

```diff
@@ -24,6 +24,7 @@ def authenticate
     @verification.generate_otp
     @verification.expire_path_token
 
+    return render plain: "success" if browser.safari?
```

Comment:
2 nits:

1. i18n the string since its user-facing
2. make this a little more informative, telling the user what to expect/what actions to take to finish verification?

Reply:
The `success` isn't really user facing in this case. The [js](https://github.com/rubygems/rubygems.org/blob/master/app/assets/javascripts/webauthn.js#L138) uses the response from this action to render a specific page.

If `success` is returned, then this view will render telling the user to close the browser.

<img width="863" alt="Screenshot 2023-06-28 at 3 51 22 PM" src="https://github.com/rubygems/rubygems.org/assets/42748004/db718a2f-8b3b-41bf-8036-2054d6e442a0">